### PR TITLE
refactor:   break the chat.jsx file into using helper.js, so we can just call the functions in chat.jsx

### LINKED ISSUE #272: [OTHER] refactor large file
### What would you like to share?

We have a large file in chat.jsx, break the file into using helper.js, so we can just call the functions in chat.jsx, if you have a better to do this don't hesitate to tell me, let's have a discussion.

### Additional information

_No response_

### 👀 Have you checked if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read and understood the rules in the [Contributing Guidelines](https://github.com/Dun-sin/Whisper/blob/main/CONTRIBUTING.md)

### Are you willing to work on this issue ?

None

### PR DESCRIPTION
**Pull request by MaybeSHAH**
# Fixes Issue

**My PR closes #272**

# 👨‍💻 Changes proposed(What did you do ?)
Keeping everything organised 
Created a class to maintain all functions of chat.jsx
import the class in chat.jsx 
and using instance of class to call functions.

# ✔️ Check List (Check all the applicable boxes)

- [] My code follows the code style of this project.
- [] This PR does not contain plagiarized content.
- [] The title and description of the PR is clear and explains the approach.

##  Note to reviewers

# 📷 Screenshots

### CODE REVIEW

**Comment by Dun-sin** [comment]
You said it yourself, this is a refactor not a fix, so please change the PR title accordingly

File: client/src/components/ChatHelper.js
Comment (Dun-sin) [inline client/src/components/ChatHelper.js:1]: please stick to the project code format and don't use classes

File: client/src/components/ChatHelper.js [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+
+    checkPartnerResponse(){
+        const currentTime = new Date().getTime();
+        const isInactive = this.lastMessageTime && 
+            currentTime - this.lastMessageTime > this.inactiveTimeThreshold;
+        if (isInactive) {
+            createBrowserNotification("your partner isn't responding, want to leave?");
+        }
+    };
+}
```
Comment (Dun-sin) [inline client/src/components/ChatHelper.js:244]: also make sure to keep an empty line at the end of each file

**Review by Dun-sin — CHANGES_REQUESTED** [review]

File: client/src/components/ChatHelper.js
Comment (MaybeSHAH) [reply client/src/components/ChatHelper.js:1]: @Dun-sin  Okay

Comment (MaybeSHAH) [reply client/src/components/ChatHelper.js:1]: without classes every functions need necessary data and dependencies as parameter and it is not good to send it eveytime while calling individual function !

Comment (Dun-sin) [reply client/src/components/ChatHelper.js:1]: also fix the merge conflict

Comment (MaybeSHAH) [reply client/src/components/ChatHelper.js:1]: @Dun-sin Yes, I am doing it.

**Pull request closed without merging**